README.md

```diff
@@ -237,7 +246,11 @@ The shared storage worklet invocation methods (`addModule`, `run`, and `selectUR
 *  The response header will only be honored if the corresponding request included the request header: `Sec-Shared-Storage-Writable: ?1`.
 *  See example usage below.
 
+## Error handling
+
+Note that the shared storage APIs may throw for several possible reasons. The following list of situations is not exhaustive, but, for example, the APIs may throw if the site invoking the API is not [enrolled](https://github.com/privacysandbox/attestation/blob/main/how-to-enroll.md) and/or [attested](https://github.com/privacysandbox/attestation/blob/main/README.md#core-privacy-attestations), if the user has disabled shared storage in site settings, if the "shared-storage" or "shared-storage-select-url" permissions policy denies access, or if one of its arguments is invalid.
 
+We encourage always wrapping calls to shared storage JS methods in `try...catch` blocks.
```

Comment:
try/catch when using async/await. If not using async/await, then they should use `.catch`. Probably best to just say that we recommend handling exceptions.

Reply:
Revised. Thoughts?